truffle/src/main/ruby/core/rubinius/api/shims/time.rb

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2014, 2015 Oracle and/or its affiliates. All rights reserved. This
+# code is released under a tri EPL/GPL/LGPL license. You can use it,
+# redistribute it and/or modify it under the terms of the:
+#
+# Eclipse Public License version 1.0
+# GNU General Public License version 2
+# GNU Lesser General Public License version 2.1
+
+class Time
+
+  def to_f
+    seconds + nsec * 0.000000001 # Truffle: optimized
```

Comment:
I don't know. I just carried it forward. Someone had modified the Rubinius file in place. All I did was move the modification out to a shim file like we've done elsewhere. It looks like @eregon originally made the change in 7e88d72dc9913801fe3bb454a96ddd4436156069.